### ISSUE TITLE
Issue building on Sierra

### ISSUE THREAD

**Issue by bababash** [paperManu/splash#32]
I'm trying to install Splash on Sierra (10.12.3) and keep getting errors. Following the instructions in the wiki, the `cmake` command completes successfully but when I run `make` this is the output I get. Not sure how to fix this?

```
[ 18%] Building CXX object src/CMakeFiles/splash-0.6.dir/world.cpp.o
clang: error: no such file or directory: 'Mapping/splash/src/../external/syphon/build/Release'
clang: warning: -framework Syphon: 'linker' input unused
make[2]: *** [src/CMakeFiles/splash-0.6.dir/world.cpp.o] Error 1
make[1]: *** [src/CMakeFiles/splash-0.6.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by paperManu** [comment]
Hi bababash,
Did you initialize the submodules of the repository?

**Comment by bababash** [comment]
Hey paperManu,

I have been following the instructions from here [https://github.com/paperManu/splash/wiki/Installation](https://github.com/paperManu/splash/wiki/Installation) word for word. Including 

>git submodule update --init

I had an issue with this part:
>brew install gstreamer1 python45

These aren't available packages but I have gstreamer and python installed (2.7 and 3.5)

Also the linux install asks for us to run
>./make_deps.sh

before running make but this isn't in the OSX instructions.
I tried with and without running make_deps.sh and still can't figure out what's wrong so I posted here.

Thanks

**Comment by paperManu** [comment]
Well, it seems that Sypon has not been built correctly. You can try forcing the build of Syphon, from the directory where you are building Splash:
```
make Syphon
```
If you get an error, then you may have your culprit.

**Comment by bababash** [comment]
Sorry paperManu that didn't do the trick and I still can't work out what I'm doing wrong :(

`[100%] Built target Syphon`

**Comment by paperManu** [comment]
I'm sorry, I just checked the documentation and found out that the OSX instructions were not complete. I fixed it, and tried it on a OSX 10.11 computer. The update instructions are there (and will be replicated to the master branch next release) : 
[Wiki installation instructions](https://github.com/paperManu/splash/wiki/Installation)

Remember that as I don't have time to support the OSX port, it may not work correctly. But if you have the time to help, I'd gladly receive pull requests.
